specification/resources/resource-manager/readme.az.md

```diff
@@ -0,0 +1,24 @@
+## AZ
+
+These settings apply only when `--az` is specified on the command line.
+
+``` yaml $(az) && $(target-mode) != 'core'
+az:
+    extensions: {svcName}
```

Comment:
Hi @wwendyc , The {svcName} should be replaced as real name.

Want to confirm whether you aim to use CLI codegen  to generate CLI commands?  core mode or extension mode?
No need to commit this file if you are not using  autorest.az to generate CLI.